src/main/java/FibonacciEven.java

```diff
@@ -0,0 +1,19 @@
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class FibonacciEven {
+
+
+    public static List<Integer> generateOnlyEvenFibonacciNumbersSum(int numberFromUser){
+        List<Integer> fibonacciNumbers = Collections.singletonList(Stream.iterate(new Integer[]{0, 1},
```

Comment:
Wyliczanie ciągu fibonacciego za pomocą streamów jest to jeden z przykładów w którym zwykły `for` czy `while` wygląda dużo lepiej - postaraj się to przepisać.